src/students/students.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@nestjs/common';
+import { Student } from '../student/student.entity';
+
+@Injectable()
+export class StudentsService {
+  async getStudents(status: number): Promise<Student[]> {
+    return await Student.find({
+      where: {
```

Comment:
patrząc na makietę (https://xd.adobe.com/view/864faeb9-d762-4277-a5d1-5b7565dcf543-d31c/screen/583bb260-3a58-4101-b33f-312170f2254b?utm_campaign=WYNIKI+TWOJEGO+ZALICZENIA+MEGAK+-+GRATULACJE&utm_medium=email&utm_source=newsletter&utm_term=2023-04-18) to ten endpoint nie musi zwraca wszystkich kolumn. Można zastosować select  i wyciągnąc tylko niezbędne info typu id, firstName, lastName itd..